celery/beat.py

```diff
@@ -381,7 +381,7 @@ def apply_async(self, entry, producer=None, advance=True, **kwargs):
 
         try:
             entry_args = [v() if isinstance(v, BeatLazyFunc) else v for v in (entry.args or [])]
-            entry_kwargs = {k: v() if isinstance(v, BeatLazyFunc) else v for k, v in entry.kwargs.items()}
+            entry_kwargs = {k: v() if isinstance(v, BeatLazyFunc) else v for k, v in (entry.kwargs.items() or {})}
```

Comment:
How can this be right?
`kwargs` is `None` so it has no `items()`.
You should check if `kwargs` is `None` and if it is call `items()` on an empty dictionary.

```suggestion
            entry_kwargs = {k: v() if isinstance(v, BeatLazyFunc) else v for k, v in (entry.kwargs or {}).items()}
```

Reply:
I wasn't fully sure so wanted more eyes for some suggestion O:) thanks